documentation/api/io.strimzi.api.kafka.model.KafkaMirrorMakerTls.adoc

```diff
@@ -1,5 +1,8 @@
-Use the `tls` property to configure TLS encryption.
-Provide a list of secrets with key names under which the certificates are stored in X.509 format.
+Use the `KafkaMirrorMakerTls` properties to configure TLS trusted certificates for connecting MirrorMaker to the cluster.
+
+.`trustedCertificates`
+Having xref:type-KafkaClientAuthenticationTls-reference[set `tls` to configure TLS encryption], use the
+`trustedCertificates` property to provide a list of secrets with key names under which the certificates are stored in X.509 format.
```

Comment:
Can we add something like this at the end?

```
If you want to enable TLS but use the default set of public certification authorities shipped with Java, you can specify the `trustedCertificates` as empty array:

.An example of enabling TLS with the default Java certificates 
[source,yaml,subs=attributes+]
----
tls:
  trustedCertificates: []
----
```

Reply:
Added